lib/jormungandr/test/integration/Main.hs

```diff
@@ -108,7 +105,7 @@ instance KnownCommand (Jormungandr n) where
     commandName = "cardano-wallet-jormungandr"
 
 main :: forall t. (t ~ Jormungandr 'Testnet) => IO ()
-main = hspec $ do
+main = hspec $ parallel $ do
```

Comment:
So we are running multiple `jörmungandr` at the same time, with random ports and random state-directories? Maybe worth clarifying what `withCardanoWalletServer` or `withServer` does.

Reply:
Yes, I fixed it. `parallel` is unlikely to work with the E2E tests.